scripts.js

```diff
@@ -14,3 +14,131 @@ close.addEventListener('click', () => {
+  },
+  project4: {
+    id: 4,
+    projectTitle: 'Profesional Art Printing Data',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Art/popup-image.png',
+    linktolive: ['See Live', 'Art/Icon-Live.svg'],
+    imageAlt: 'Popup project image',
+    linktosource: ['See Source', 'Art/Icon-Source.svg'],
+  },
+  project5: {
+    id: 5,
+    projectTitle: 'Data Dashboard Healthcare',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Art/popup-image.png',
+    linktolive: ['See Live', 'Art/Icon-Live.svg'],
+    imageAlt: 'Popup project image',
+    linktosource: ['See Source', 'Art/Icon-Source.svg'],
+  },
+  project6: {
+    id: 6,
+    projectTitle: 'Website Protfolio',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    image: 'Art/popup-image.png',
+    imageAlt: 'Popup project image',
+    linktolive: ['See Live', 'Art/Icon-Live.svg'],
+    linktosource: ['See Source', 'Art/Icon-Source.svg'],
+  },
+};
+
+const tags = {
+  tagHtml: 'html',
+  tagBootstrap: 'Bootstrap',
+  tagRubyOnRails: 'Ruby on rails',
+};
+
+const popupProjectDetails = document.querySelectorAll('.seeProjectButton');
+const popupProjectMobile = document.querySelector('.project-popup-mobile');
+const bgBlur = document.querySelector('.works');
+const lockScroll = document.querySelector('.content');
+
+function displayProject1(event) {
+  popupProjectMobile.classList.remove('popup');
+  bgBlur.classList.add('blur');
+  lockScroll.classList.add('lockScroll');
+  document.querySelector('.toolbar').classList.add('blur');
+  document.querySelector('.headline').classList.add('blur');
+  document.querySelector('.aboutme').classList.add('blur');
+
+  let selectedProjects;
+
+  Object.keys(projects).forEach((project) => {
+    if (event.target.id === projects[project].id.toString()) {
+      selectedProjects = projects[project];
+    }
+  });
+
+  popupProjectMobile.innerHTML = `
+  <div class="popup-top" id="popup-top">
+    <h4 class="popup-header" id="popupHeader">${selectedProjects.projectTitle}</h4>
```

Comment:
- [ ] In mobile view (starts at 425 px), the bottom of popup modal is not visible and the buttons seem to get out of the popup modal bottom border, kindly address this issue:
<img width="321" alt="Screenshot 2022-04-12 at 17 03 50" src="https://user-images.githubusercontent.com/77038610/162968656-461690b6-8281-4a44-b04c-3d66f7c07dc6.png">

- [ ] Please make sure that the content inside buttons does not wrap and is responsive.